app/import_functions.go

```diff
@@ -1004,9 +1004,9 @@ func (a *App) importReplies(data []ReplyImportData, post *model.Post, teamId str
 		user := users[*replyData.User]
 
 		// Check if this post already exists.
-		replies, err := a.Srv().Store.Post().GetPostsCreatedAt(post.ChannelId, *replyData.CreateAt)
-		if err != nil {
-			return err
+		replies, nErr := a.Srv().Store.Post().GetPostsCreatedAt(post.ChannelId, *replyData.CreateAt)
+		if nErr != nil {
+			return model.NewAppError("importReplies", "app.post.get_posts_created_att.app_error", nil, nErr.Error(), http.StatusInternalServerError)
```

Comment:
This should be "app.post.get_posts_created_at"

Reply:
Fixed